helpers/browser/driver.js

```diff
@@ -110,12 +116,17 @@ class ChromeProtocol {
         if (err) {
           return reject(result);
         }
-
         resolve(result);
       });
     });
   }
 
+  pendingCommandsComplete() {
```

Comment:
This is really interesting, so chrome once ready? fires only when all commands are complete?